azerothcore/azerothcore-wotlk
# fix(DB): Outrunner Alarion gossip menu

## Changes Proposed:
-  Added conditions for gossip

## Issues Addressed:

- Closes #6389

## SOURCE:


## Tests Performed:

- Tested in-game with success.

## How to Test the Changes:


1. .go c id 15301
2. Gossip text: "_If you're heading toward the city, then I have something important for you to do!_"
3. .quest add 8350
4. .quest reward 8350
5. Gossip text: "_Ah, <name> - it is good to see you again. I trust your affairs go well._"

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
